app/javascript/v3/views/onboarding/InviteTeam.vue

```diff
@@ -0,0 +1,142 @@
+        :class="{
+          'border-dashed grid place-content-center':
+            emailsToInvite.length === 0,
+        }"
+      >
+        <div v-if="emailsToInvite.length === 0" class="px-5 py-16">
+          <p class="text-sm text-slate-400">
+            {{ $t('START_ONBOARDING.INVITE_TEAM.LIST_EMPTY') }}
+          </p>
+        </div>
+        <div
+          v-for="email in emailsToInvite"
+          :key="email"
+          class="flex items-center justify-between p-4"
+        >
+          <div class="flex items-center gap-2">
+            <woot-thumbnail :username="email" size="24px" />
+            <span
+              class="text-sm font-medium text-slate-700 dark:text-slate-200"
+            >
+              {{ email }}
+            </span>
+          </div>
+          <woot-button
+            variant="clear"
+            size="small"
+            color-scheme="secondary"
+            icon="dismiss"
+            @click="emailsToInvite.splice(emailsToInvite.indexOf(email), 1)"
+          />
+        </div>
+      </div>
+      <div class="space-y-2">
+        <submit-button
+          button-class="flex justify-center w-full text-sm text-center"
+          :button-text="$t('START_ONBOARDING.INVITE_TEAM.SUBMIT.BUTTON')"
+          @click="onSubmit"
+        />
+
+        <woot-button
+          variant="clear"
+          color-scheme="secondary"
+          size="expanded"
+          class="w-full"
+          is-expanded
+          @click="skipToFoundersNote"
+        >
+          {{ $t('START_ONBOARDING.INVITE_TEAM.SKIP') }}
+        </woot-button>
+      </div>
+    </section>
+  </onboarding-base-modal>
+</template>
+
+<script>
+import FormInput from 'v3/components/Form/Input.vue';
+import SubmitButton from 'dashboard/components/buttons/FormSubmitButton.vue';
+import OnboardingBaseModal from 'v3/views/onboarding/BaseModal.vue';
+import emailValidator from 'vuelidate/lib/validators/email';
+
+import AgentAPI from 'dashboard/api/agents.js';
+import alertMixin from 'shared/mixins/alertMixin';
+
+export default {
+  components: {
+    SubmitButton,
+    FormInput,
+    OnboardingBaseModal,
+  },
+  mixins: [alertMixin],
+  data() {
+    return {
+      emailsToInvite: [],
+      emailToAdd: '',
+    };
+  },
+  methods: {
+    pushEmail() {
+      if (!this.emailToAdd) return;
+      if (emailValidator(this.emailToAdd)) {
+        this.emailsToInvite.push(this.emailToAdd);
+        this.emailsToInvite = [...new Set(this.emailsToInvite)];
+        this.emailToAdd = '';
+      } else {
+        this.showAlert(this.$t('START_ONBOARDING.INVITE_TEAM.EMAIL_ERROR'));
+      }
+    },
+    async skipToFoundersNote() {
+      try {
+        await AgentAPI.bulkInvite({ emails: [] });
+        this.$router.push({ name: 'onboarding_founders_note' });
+      } catch (error) {
+        this.showAlert(this.$t('START_ONBOARDING.INVITE_TEAM.SUBMIT.ERROR'));
+      }
+      this.$router.push({ name: 'onboarding_founders_note' });
```

Comment:
Redundant navigation call after the try-catch block.

Reply:
Yeah, seems like this block was doing nothing. Removed it. Also disabled founders note for non cloud instances